### DISCUSSION TITLE
wrong direction of official ".minecraft" on Arch Linux

### DISCUSSION THREAD

**Discussion by FTS427** [Steve-xmh/scl#96] [Q&A]
When change the game folder to "Official game folder", SCL may not use ".minecraft" under "/home/my_username/", and I cann't find where the "Official game folder" is.
Can anyone help me?

Sorry for my poor English

**Comment by Steve-xmh** [comment]
SCL 在 Unix/macOS 上使用 `dirs::home_dir()` 获取用户的主目录，并在其下创建 `.minecraft` 文件夹作为游戏文件夹。

如果函数调用失败，就会换成一个硬编码了的 `~/.minecraft` 文件夹。

https://github.com/Steve-xmh/scl/blob/0c0be1dc67e94c5fbb5acfe540f53aceee7cdb79/scl-core/src/path.rs#L16-L24
